libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/InsightDrillConfigPanel/useInsightDrillConfigPanel.ts

```diff
@@ -200,7 +200,7 @@ export const useInsightDrillConfigPanel = (props: IUseDrillConfigPanelProps) =>
                 dispatch(
                     addDrillDownForInsightWidget(
                         widgetRef,
-                        idRef(attributeDescriptor!.attributeHeader.identifier, "displayForm"),
+                        idRef(attributeDescriptor!.attributeHeader.formOf.identifier, "displayForm"),
```

Comment:
Please check if this is correct -- having attribute id and passing "displayForm" as idref type? Looks suspicious.

Reply:
yes, it looks like inconsistent but we need it as workaround for the situation that the reference is saved in `ignoredDrillDownHierarchies` property in widget. when we save the widget, the backend just accepts the label only and we need to compare the reference with the attributes in the hierarchies.

Reply:
in that case, please add some short comment to the code explaining the situation so that it does not confuse others

Reply:
I think that we should not do this. It should be type "attribute". Can we found out how complicated it is to change this on backend so it is allowed?

Reply:
I'm working on backend change to allow the attribute support. I will let you know if I managed to put it together.

Reply:
@phong-nguyen-duy Good news! I was able to fix it on backend, thanks to Dan and Pepi 🎉
The change is deployed on https://staging.dev-latest.stg11.panther.intgdc.com/

I was able to modify my dashboard via API call to setup ignored hierarchies for an attribute:

```
"ignoredDrillDownHierarchies": [
    {
        "type": "attributeHierarchyReference",
        "attributeHierarchy": "firstHierarchy",
        "attribute": {
            "identifier": {
                "id": "customers.state",
                "type": "attribute"
            }
        }
    }
],
```

There's longer a reason for the label with attribute id hack and you can deliver the feature without compromise :)

Reply:
fixed